Client/mods/deathmatch/logic/luadefs/CLuaBuildingDefs.cpp

```diff
@@ -63,12 +63,14 @@ CClientBuilding* CLuaBuildingDefs::CreateBuilding(lua_State* const luaVM, std::u
     return pBuilding;
 }
 
+// Deprecated
 void CLuaBuildingDefs::RemoveAllGameBuildings()
 {
-    m_pBuildingManager->RemoveAllGameBuildings();
+    CLuaWorldDefs::RemoveGameWorld();
 }
 
+// Deprecated
 void CLuaBuildingDefs::RestoreGameBuildings()
 {
-    m_pBuildingManager->RestoreAllGameBuildings();
+    CLuaWorldDefs::RestoreGameWorld();
 }
```

Comment:
Please add the deprecated functions to `clientDeprecatedList` in `CResourceChecker.Data.h`